config/mainnet.json

```diff
@@ -233,13 +233,18 @@
   "adminACLV0Contracts": [
     {
       "address": "0x39F0e1e8Df2C135Df35B3FA496e943ac4B90Fe0f",
-      "name": "Admin ACL V0 for V3 core",
+      "name": "(deprecated) Admin ACL V0 for V3 core",
       "startBlock": 15726705
     },
     {
       "address": "0x286bA5808A98c9237449182E05740E4B7284389e",
-      "name": "Admin ACL V0 for Explorations",
+      "name": "(deprecates) Admin ACL V0 for Explorations",
       "startBlock": 15791055
+    },
+    {
+      "address": "0x286bA5808A98c9237449182E05740E4B7284389e",
```

Comment:
This should be `0x18b18cF97a3D8BCC43f8D1Df282CebA85f717C82` IIUC.

Reply:
oof - good catch 😮‍💨 🥇 fixed!

Reply:
huzzah! 🙏 :vibe: